frontend/src/metabase/visualizations/components/TransformedVisualization/TransformedVisualization.tsx

```diff
@@ -31,7 +31,7 @@ export const TransformedVisualization = ({
 
   const transformedSeries = useMemo(() => {
     return transformSeries(rawSeries, settings, renderingContext);
-  }, [transformSeries, rawSeries, settings]);
+  }, [transformSeries, rawSeries, settings, renderingContext]);
```

Comment:
This is not related to the rest of the PR, but I added it because I saw it was missing.